java-api.md

```diff
@@ -28,6 +28,19 @@ This will result in generator always generating value 4.
 Almost all methods support all Java primitive number types (byte, short, int, long, float, double), there is no need for limiting only on int, long or double types if other types are more suitable in particular case.
 Below is given a list of helper methods for construction of `ObjectGenerator`. Methods are provided by `BuilderMethods` class.
 
+## Primitive
```

Comment:
'primitive' sounds as a generator of values of primitive data types. Here we have constant values. Is this the clear naming for the generator?